pkgs/development/python-modules/pyevtk/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  pytest-runner,
+  numpy,
+  pytestCheckHook,
+  pytest-cov
+}:
+
+buildPythonPackage rec {
+  pname = "pyevtk";
+  version = "1.2.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "pyscience-projects";
+    repo = "pyevtk";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-HrodoVxjREZiutgRJ3ZUrART29+gAZfpR9f4A4SRh4Q=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [ pytest-runner numpy ];
+
+  pythonImportsCheck = [ "pyevtk" ];
+  nativeCheckInputs = [ pytestCheckHook pytest-cov ];
```

```suggestion
  build-system = [ setuptools ];

  dependencies = [ numpy ];

  nativeCheckInputs = [ pytestCheckHook pytest-cov ];

  pythonImportsCheck = [ "pyevtk" ];
```
those deps are rather useless in nixpkgs